### PR TITLE
Do not display 'Why not start the conversation?' for specialNarrow and searchNarrow

### PR DESCRIPTION
**Pull request by agzuniverse**
This subtitle does not seem to make sense for specialNarrow and searchNarrow as one cannot start a conversation directly from these.

### CODE REVIEW

**Comment by borisyankov** [comment]
Look for some logic that we already have, telling you 'you can not send anything to this narrow' and reuse.

**Comment by agzuniverse** [comment]
Updated, now it uses `canSendToNarrow`.

**Comment by borisyankov** [comment]
Cool, that is what I meant 👍

File: src/message/NoMessages.js [hunk 1]
```diff
@@ -64,7 +65,7 @@ class NoMessages extends PureComponent<Props> {
     return (
       <View style={styles.container}>
         <Label style={styles.text} text={message.text} />
-        <Label text="Why not start the conversation?" />
+        {canSendToNarrow(narrow) ? <Label text="Why not start the conversation?" /> : null}
```
Comment (jainkuniya) [inline src/message/NoMessages.js]: I guess
`{canSendToNarrow(narrow) && <Label text="Why not start the conversation?" />}`
would be better.

Comment (borisyankov) [reply src/message/NoMessages.js]: Yeah, I also prefer what you suggest, the only negative is that if we mistakenly are returning a `string` or `number` it tries to render it. I suspect `eslint` should have a rule for that, but I don't think they do.

**Pull request merged**